src/app/pages/RewardPage/hooks/useGetFeesEarnedClaimAmount.ts

```diff
@@ -1,17 +1,142 @@
-import { useAccount } from 'app/hooks/useAccount';
-import { useCacheCallWithValue } from 'app/hooks/useCacheCallWithValue';
+import { useEffect, useState, useMemo, useCallback } from 'react';
+import { useAccount, useBlockSync } from 'app/hooks/useAccount';
+import { bridgeNetwork } from 'app/pages/BridgeDepositPage/utils/bridge-network';
+import { Asset, Chain } from 'types';
 import { getContract } from 'utils/blockchain/contract-helpers';
+import { useSelector } from 'react-redux';
+import { selectWalletProvider } from 'app/containers/WalletProvider/selectors';
+import { fixNumber } from 'utils/helpers';
+import { bignumber } from 'mathjs';
+import { AssetsDictionary } from 'utils/dictionaries/assets-dictionary';
+export interface IEarnedFee {
+  asset: Asset;
+  contract: string;
```

Comment:
Again, this is not a contract but a contract address. Rename it to either `contractAddress` or `assetContractAddress`.